https/README.md

```diff
@@ -0,0 +1,24 @@
+# HTTPS Package for Prometheus
+
+The `https` directory contains a Go package and a sample configuration file for running `node_exporter` with HTTPS instead of HTTP.
+When running a server with TLS use the flag `--web.config`
+
+e.g. `./node_exporter --web.config="web-config.yml"`
+If the config is kept within the https directory.
```

Comment:
Wouldn't it be https/web-config.yml then?

Reply:
I'm going to merge this and we can do more work in additional MRs.